spec/lib/programme_to_iati_status_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+RSpec.describe "#programme_status_to_iati_status" do
+  context "when the user sets the programme status for an activity" do
+    it "the correct iati status will be set automatically" do
+      status = ProgrammeToIatiStatus.new.programme_status_to_iati_status("01")
+      expect(status).to eq "2"
+
+      status = ProgrammeToIatiStatus.new.programme_status_to_iati_status("02")
```

Comment:
I'm not sure if testing all the possible inputs and outputs is necessary here.

Reply:
I thought it was valuable to have some representation of the required mappings in the test suite. Part of the motivation for extracting this logic into its own class was so we could test that conversion without going through the whole workflow for each variation.

Reply:
Shall I leave just one or two then? I never know how many tests are too many tests O:-)

Reply:
Personally I would rather have all the mapping cases tested. These test will be fast to run and I don't expect them to cause much maintenance overhead.

Reply:
Yes, I think I was typing my last message without having seen yours James. I'll leave it as it is :-)